src/cli/index.js

```diff
@@ -58,6 +58,7 @@ exports.run = (opts = {}) => {
             "--update-refs",
             'update screenshot references or gather them if they do not exist ("assertView" command)',
         )
+        .option("--run-failed", "only run tests that failed during the last run")
```

Comment:
I think it may be named something like `--run-last-failed`, that would sound more clear